rfcs/0000-windows-containers.md

```diff
@@ -0,0 +1,76 @@
+## Problem:
+
+kpack does not support building containers that can run on windows
+
+## Background
+
+kpack has an [rfc](https://github.com/pivotal/kpack/pull/534) to add windows support
```

Comment:
isn't this just linking to this pull request

Reply:
😂